kubeflow/common/prototypes/centraldashboard.jsonnet

```diff
@@ -3,7 +3,7 @@
 // @description centraldashboard Component
 // @shortDescription centraldashboard
 // @param name string Name
-// @optionalParam image string gcr.io/kubeflow-images-public/centraldashboard:v0.5.0 Image for the central dashboard
+// @optionalParam image string gcr.io/kubeflow-images-public/centraldashboard:v20190613-v0.4.0-rc.1-527-g7ca3cfb3 Image for the central dashboard
```

Comment:
Can we retag the image that's being pointed to here so it doesn't appear that it's a stepping backwards?

Reply:
See https://github.com/kubeflow/kubeflow/issues/2997

There are two issues here

1. The tags of the master branch
1. The expression used in the Makefile to get the descriptor; I think we want to switch to using

    ```
   git describe --tags --long
    ```

Reply:
From the command above I am getting: `v0.4.0-rc.1-528-g65c0b34f`. Is this correct?